Remove relative path so -I$PETSC_DIR/include finds installed headers

Index: petsc-3.4.2/src/mat/examples/tests/ex110.c
===================================================================
--- petsc-3.4.2.orig/src/mat/examples/tests/ex110.c	2013-08-20 15:26:51.000000000 +0200
+++ petsc-3.4.2/src/mat/examples/tests/ex110.c	2013-08-20 15:26:51.000000000 +0200
@@ -1,7 +1,7 @@
 static char help[] = "Testing MatCreateMPIAIJWithSplitArrays().\n\n";
 
 #include <petscmat.h>
-#include <../src/mat/impls/aij/mpi/mpiaij.h>
+#include <src/mat/impls/aij/mpi/mpiaij.h>
 
 #undef __FUNCT__
 #define __FUNCT__ "main"
Index: petsc-3.4.2/src/mat/examples/tutorials/ex4.c
===================================================================
--- petsc-3.4.2.orig/src/mat/examples/tutorials/ex4.c	2013-08-20 15:26:51.000000000 +0200
+++ petsc-3.4.2/src/mat/examples/tutorials/ex4.c	2013-08-20 15:26:51.000000000 +0200
@@ -89,7 +89,7 @@
   return 0;
 }
 
-#include <../src/mat/impls/dense/mpi/mpidense.h>
+#include <src/mat/impls/dense/mpi/mpidense.h>
 #include <petsc-private/vecimpl.h>
 
 #undef __FUNCT__
Index: petsc-3.4.2/src/mat/examples/tutorials/ex5.c
===================================================================
--- petsc-3.4.2.orig/src/mat/examples/tutorials/ex5.c	2013-08-20 15:26:51.000000000 +0200
+++ petsc-3.4.2/src/mat/examples/tutorials/ex5.c	2013-08-20 15:26:51.000000000 +0200
@@ -7,7 +7,7 @@
 #include <petscviewer.h>
 
 #if defined(USE_FAST_MAT_SET_VALUES)
-#include <../src/mat/impls/aij/mpi/mpiaij.h>
+#include <src/mat/impls/aij/mpi/mpiaij.h>
 #define MatSetValues MatSetValues_MPIAIJ
 #else
 #include <petscmat.h>
